optimum/habana/accelerate/accelerator.py

```diff
@@ -176,6 +181,7 @@ def __init__(
         self.scaler_handler = None
         self.init_handler = None
         self.fp8_recipe_handler = None
+        self.fp8_recipe_format = None
```

Comment:
Why are we resetting the fp8_recipe_format here?

Reply:
we want to user to have option to select the fp8_recipe_format